stagemonitor-core/src/main/java/org/stagemonitor/core/CorePlugin.java

```diff
@@ -396,6 +396,44 @@
 			.configurationCategory(CORE_PLUGIN_NAME)
 			.tags("elasticsearch", "advanced")
 			.buildWithDefault(5);
+	private final ConfigurationOption<Boolean> enableSpringCloudConfigConfigurationSource = ConfigurationOption.booleanOption()
```

Comment:
I think this is redundant. Just check if `stagemonitor.configuration.springcloud.address` is set.

Reply:
done